Source/WebCore/dom/CustomElementReactionQueue.cpp

```diff
@@ -386,6 +386,18 @@ inline void CustomElementQueue::processQueue(JSC::JSGlobalObject* state)
     }
 }
 
+Vector<GCReachableRef<Element>> CustomElementQueue::takeElements()
+{
+    RELEASE_ASSERT(!m_invoking);
+    return WTFMove(m_elements);
```

Comment:
I like std::exchange() better in such cases. The reasoning is that it is normally not safe to assume `WTFMove()` leaves m_elements in a good state (though it does here because of the way we wrote the Vector move constructor).

Reply:
Ok, sure.